pyproject.toml

```diff
@@ -5,17 +5,17 @@ description = "CommCare Sync simplifies the management of your CommCare data pip
 readme = "README.md"
 requires-python = "~=3.10.0"  # TODO: ">=3.10,<3.15"
 dependencies = [
-    "Django",
+    "Django~=3.2",
```

Comment:
:exclamation: That's old, and past EOL

Reply:
This was not meant as a criticism of your work. I hope it didn't come across that way.

Reply:
No, not at all. My exclamation point wasn't annoyance, it was strong agreement. Coming back to this codebase has been a lesson in scheduling infrequent but regular maintenance for any codebase that one hasn't already thrown away.